### PR TITLE
JSDocs comments and Design content

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-set-up-docs-tools-080920](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-set-up-docs-tools-080920)

File: src/app/docs/split-view/split-view-docs.component.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use split views when users are likely to interact with a small number of items in the list. Use regular lists instead, and follow the <a href="https://developer.blackbaud.com/skyux/components/flyout#ux-guidelines">flyout guidelines</a> to determine whether to use flyouts or navigate to selected records.
```
Comment (blackbaud-johnly) [inline src/app/docs/split-view/split-view-docs.component.html:39]: Should we update this guidance to recommend data managers instead of regular lists?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-set-up-docs-tools-080920](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-set-up-docs-tools-080920)

File: src/app/public/modules/split-view/split-view.component.ts [hunk 2]
```diff
@@ -74,13 +78,21 @@ import {
 })
 export class SkySplitViewComponent implements OnInit, OnDestroy {
 
+  /**
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/public/modules/split-view/split-view.component.ts:81]: This is missing the default value. The current docs show the default value as `Back to list`.

File: src/app/public/modules/split-view/split-view-drawer.component.ts [hunk 3]
```diff
@@ -44,9 +47,15 @@ let skySplitViewNextId = 0;
 })
 export class SkySplitViewDrawerComponent implements AfterViewInit, OnInit, OnDestroy {
 
+  /**
+   * Sets the list panel's `aria-label` attribute to support accessibility.
+   */
   @Input()
   public ariaLabel: string;
 
+  /**
```
Comment (Blackbaud-DeniseCisneros) [inline src/app/public/modules/split-view/split-view-drawer.component.ts:56]: This is missing the default value. The current docs show the default value as `320`.

File: src/app/docs/split-view/split-view-docs.component.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/docs/split-view/split-view-docs.component.html:39]: @Blackbaud-SteveBrush, is this something to look into? Should I file an issue or post a question to the team Slack channel? Let me know if you think this is worth addressing before we merge these JSDoc comments.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-set-up-docs-tools-080920](https://developer.blackbaud.com/skyux-split-view?_sp.spa=pr-set-up-docs-tools-080920)

File: src/app/docs/split-view/split-view-docs.component.html (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/docs/split-view/split-view-docs.component.html:39]: Let's keep it as-is for now. A new issue would be a good idea. Thanks!

**Review by Blackbaud-SteveBrush — APPROVED** [review]

File: src/app/docs/split-view/split-view-docs.component.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/docs/split-view/split-view-docs.component.html:39]: I created https://github.com/blackbaud/skyux-split-view/issues/38 to capture this.

**Pull request merged**